Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.4-slim
+
+WORKDIR /app
+
+COPY Gemfile .
+
+RUN apt-get -y update && apt-get -y install python3 python3-pip make gcc  g++ git
+
+RUN pip3 install ruamel.yaml
+
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+RUN gem install jekyll -v3.8
```

Comment:
```suggestion
RUN gem install jekyll -v3.8
```
This step is unsed.

Reply:
I used this line to fix the version of Jekyll, to avoid all updates and breaking changes of Jekyll 4+. Unless your code is compatible with Jekyll 4.0 ?

Reply:
Check this: <https://github.com/cotes2020/jekyll-theme-chirpy/blob/cf6c94a3845a0c4911e2eaf6490699ce31988a86/Gemfile#L3>

Gemfile already indicated the Jekyll version, so the line `RUN gem install jekyll -v3.8` won't work.

Reply:
Oh, you're right.
I thought your Gemfile was installing 4.0, my memory is a bit blurry (I built this image a month ago https://hub.docker.com/r/blafy/chirpy-build).
I will fix the dockerfile then :)

Reply:
One more thing, I build the image locally (on mac) and found the size  up to **834 MB**, could you reduce the size?

```console
$ docker image ls
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
chirpy-pr           latest              582049aa1b5f        19 hours ago        834MB
```